src/tests/ForecastSummaries.test.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
```

Comment:
This test is currently failing for several reasons:
- The snapshot is out of date and needs to be updated
- The icon property is expected to be of type Number and not String (see the proptypes of the component under test for proof)
- You are not passing all required props. The function onForecastSelect needs mocking and passing in

Your valid props object should something like this:
```
const validProps = {
    forecasts: [
      {
        date: 1111111,
        description: 'Stub description 1',
        icon: 800,
        temperature: {
          max: 22,
          min: 12,
        },
      },
      {
        date: 2222222,
        description: 'stub description2',
        icon: 602,
        temperature: {
          max: 24,
          min: 13,
        },
      },
    ],
    onForecastSelect: jest.fn(),
  };
```

You will then need to update the props being passed into all instances of the component being created in the tests in order to make them all pass